File: 00_intro_to_c/agree.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    char answer = get_char("Agree y/n? "):

    if (answer == 'y' || answer == 'Y')
    {
        printf("Agreed\n");
    }
    else if (answer == 'n' || answer == 'N')
    {
        printf("not agreed\n");
    }
}

File: 05_python/agree.py
from cs50 import get_string

# Prompt user to agree
s = get_string("Agree y/n? ")

# Check whether agreed
if s == "Y" or s == "y":
    print("Agreed")
elif s == "N" or s == "n":
    print("Not agreed")

# oop
if s.lower() in ["y", "yes"]